apollo-router/src/error.rs

```diff
@@ -136,10 +137,19 @@ impl FetchError {
 #[serde(rename_all = "camelCase")]
 pub struct Location {
```

Comment:
Why don't we just use the `router_bridge` Location here? Is it because of the serde rename requirement? We could probably put that into the router bridge and it would be fine.

If we did just use the same type, then it would help avoid errors such as you caught just below.